### PR TITLE
Add INSTALL_NERDCTL option and create "rd" edition

### PR DESCRIPTION
**Pull request by jandubois**
Adds buildctl, buildkitd, and nerdctl binaries, and marks them all as setuid "root".

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,8 +18,11 @@ mkimage:
 		.
 
 .PHONY: iso
-iso:
-	ALPINE_VERSION=$(ALPINE_VERSION) REPO_VERSION=$(REPO_VERSION) EDITION=$(EDITION) ./build.sh
+iso: nerdctl-$(NERDCTL_VERSION)
+	ALPINE_VERSION=$(ALPINE_VERSION) NERDCTL_VERSION=$(NERDCTL_VERSION) REPO_VERSION=$(REPO_VERSION) EDITION=$(EDITION) ./build.sh
+
+nerdctl-$(NERDCTL_VERSION):
+	curl -o $@ -Ls https://github.com/containerd/nerdctl/releases/download/v$(NERDCTL_VERSION)/nerdctl-full-$(NERDCTL_VERSION)-linux-amd64.tar.gz
```
Comment (AkihiroSuda) [inline Makefile:25]: This package currently contains dynamically-linked binaries and does not work on Alpine.

`apk add libc6-compat` doesn't seem enough

```console
# ./containerd
Error relocating ./containerd: __fprintf_chk: symbol not found
Error relocating ./containerd: __vfprintf_chk: symbol not found
```

**Comment by AkihiroSuda** [comment]
I think it's more consistent to have the code on Lima side, but not a strong opinion.
(We can extend the yaml to support specifying nerdctl binary archive, setuid mode, etc.)

File: Makefile (hunk 1 above)
Comment (jandubois) [reply Makefile:25]: Note that we will be using the `containerd` from `k3s`, so we really only going to run `buildctl`, `buildkitd`, and `nerdctl`. Nothing else even gets copied into the image.

Comment (jandubois) [reply Makefile:25]: Turns out we also need the `libexec` subtree... 😄

**Comment by jandubois** [comment]
> I think it's more consistent to have the code on Lima side, but not a strong opinion.
> (We can extend the yaml to support specifying nerdctl binary archive, setuid mode, etc.)

Yeah, we can. But for now this is just experimental, so having it in the ISO is fine, I think.

I've created a separate `rd` edition of the image, to avoid polluting the regular images. Also image size has grown from 47M to 119M (after adding `libexec/cni`). 😢

**Review by mattfarina — APPROVED** [review]

**Comment by AkihiroSuda** [comment]
> cni 

You probably only need "bridge", "portmap", "firewall", "tuning", “isolation”

**Comment by jandubois** [comment]
> You probably only need "bridge", "portmap", "firewall", "tuning", “isolation”

Thanks! We'll fine-tune this later; for now I just wanted to provide an image for other to experiment with.

We may still want to with your suggestion about letting lima handle the installation, but I'm not sure if I like the added complexity in the `lima.yaml` for this single use case.

**Comment by jandubois** [comment]
> > You probably only need "bridge", "portmap", "firewall", "tuning", “isolation”
> 
> Thanks! We'll fine-tune this later; for now I just wanted to provide an image for other to experiment with.

I've changed my mind on this; it will be hard to remove plugins later, as we won't know if it breaks anybody's use case. So I'll start with your minimal list (plus `host-local`, which will be required to run Rancher in Rancher Desktop 😄 ): https://github.com/lima-vm/alpine-lima/pull/15

**Pull request merged**